packages/react/src/icons/some/IconDiscord.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { IconProps } from '../Icon.interface';
+import classNames from '../../utils/classNames';
+import styles from '../Icon.module.css';
+
+const IconDiscord: React.FC<IconProps> = ({ size = 's', className = '', style = {}, ...rest }: IconProps) => (
+  <svg
+    className={classNames(styles.icon, styles[size], className)}
+    style={style}
+    viewBox="0 0 24 24"
+    xmlns="http://www.w3.org/2000/svg"
+    {...rest}
+  >
+    <g fill="none" fillRule="evenodd">
+      <path d="M0 0h24v24H0z" />
+      <path
+        d="M19.071 3a1.93 1.93 0 011.923 1.778l.006.15v14.143a1.93 1.93 0 01-1.778 1.923l-.15.006H4.928a1.93 1.93 0 01-1.923-1.778L3 19.072V4.928a1.93 1.93 0 011.778-1.923L4.928 3h14.143zm-5.429 5l-.144.165.223.07c1.595.53 2.338 1.236 2.338 1.236a8.385 8.385 0 00-5.173-.967l-.088.007-.087.014-.285.034a8.05 8.05 0 00-1.815.514l-.235.1-.326.155-.28.143.08-.07c.232-.191.93-.708 2.153-1.129l.235-.077.236-.071L10.371 8l-.146.005-.074.005c-.425.037-1.414.202-2.471.927l-.199.143-.122.24-.146.313c-.408.91-1.145 2.852-1.209 5.13L6 15.078l.059.088.033.047c.264.352 1.11 1.28 2.798 1.414l.247.014.69-.854-.2-.065c-.9-.319-1.369-.836-1.532-1.05l-.069-.098.131.083.157.092.017.015.117.067.256.134a8.634 8.634 0 002.006.658 7.205 7.205 0 002.665.01c.29-.05.584-.12.883-.217l.302-.104.527-.209a5.88 5.88 0 00.712-.362l.25-.157-.036.052-.03.041c-.165.215-.65.75-1.588 1.067l-.208.064.68.833.246-.014c2.09-.166 2.9-1.55 2.9-1.55 0-3.311-1.48-5.996-1.48-5.996-1.064-.797-2.09-1.007-2.584-1.061l-.16-.014L13.642 8zm-3.559 3.837c.587 0 1.06.514 1.05 1.141 0 .628-.463 1.142-1.05 1.142-.576 0-1.049-.514-1.049-1.142 0-.627.463-1.141 1.05-1.141zm3.755 0c.586 0 1.049.514 1.049 1.141 0 .628-.463 1.142-1.05 1.142-.575 0-1.048-.514-1.048-1.142 0-.627.462-1.141 1.049-1.141z"
+        fill="currentColor"
+      />
+    </g>
```

Comment:
I'm wondering would it be possible to use the SVG files in core here. That way you wouldn't have to maintain the SVG content in two places. Not sure if there's a real problem here, so not a showstopper to me.

Reply:
I added the new icons using a script. The script could maybe be tweaked a bit, so that it could be used to create new icons similarly to the `scaffold` script we have for creating components.